### ISSUE TITLE
DIFFERENCE function is not working

### ISSUE THREAD

**Issue by KannarFr** [cityzendata/warp10-platform#302]
Hi,

It seems there is a problem with DIFFERENCE function: example doesn't work well [here](http://www.warp10.io/reference/functions/function_DIFFERENCE/) and I can't use it well in my warpscript.

KannarFR.

**Comment by KannarFr** [comment]
```
// Converts the two LISTS into sets
[ 'foo' 'bar' ] ->SET
[ 'foo' 'bar' 'bob' ] ->SET
// Compute the difference
DIFFERENCE
// convert back the SET into LIST
SET->
```

It seems swap list make it works, strange behavior.

```
// Converts the two LISTS into sets
[ 'foo' 'bar'  'bob' ] ->SET
[ 'foo' 'bar' ] ->SET
// Compute the difference
DIFFERENCE
// convert back the SET into LIST
SET->
```

Is working!

**Comment by KannarFr** [comment]
Oh I get the point it's normal... Sorry.

**Comment by KannarFr** [comment]
I tihnk the example is confusing, you should swap lines to illustrate your point. WDYT?

**Comment by hbs** [comment]
A bug was recently corrected in DIFFERENCE see #275

**Comment by hbs** [comment]
The doc has been updated, thanks for the head up.
